src/core/index.js

```diff
@@ -11,7 +11,7 @@ import initGlobalAPI from './global-api';
  * @returns {Number|void} If the page is already laoded returns the result of the setTimeout callback,
  *  otherwise it only attaches the callback to the DOMContentLoaded event
  */
-function ready(callback) {
+export function documentReady(callback) {
```

Comment:
Is it exported for tests ?

Reply:
Yeah, otherwise not imported by anything else currently.

Reply:
But maybe `documentReady` is better in a separate util file.

Reply:
I think the current file seems ok as it is not really a utility right